android/composeui/src/main/java/com/stadiamaps/ferrostar/composeui/config/NavigationViewComponentBuilder.kt

```diff
@@ -19,14 +19,16 @@ import com.stadiamaps.ferrostar.composeui.views.components.TripProgressView
 import com.stadiamaps.ferrostar.core.NavigationUiState
 
 data class NavigationViewComponentBuilder(
-    val instructionsView: @Composable (modifier: Modifier, uiState: NavigationUiState) -> Unit,
-    val progressView:
+    internal val instructionsView:
```

Comment:
This avoids users accidentally trying to do: 

```kotlin
NavigationViewComponentBuilder.Default()
    .instructionsView { }
```

Which is very easy to do and highly problematic.